app/pages/GoodsTooLargeForContainerYesNoPage.scala

```diff
@@ -19,7 +19,7 @@ package pages
 import pages.sections.OtherQuestionsSection
 import play.api.libs.json.JsPath
 
-case object GoodsTooLargeForContainerYesNoPage extends QuestionPage[Boolean] {
+case class GoodsTooLargeForContainerYesNoPage(messageId: String) extends QuestionPage[Boolean] {
```

Comment:
What's the rationale behind adding message ID here?

Reply:
removed